tests_e2e/tests/ext_sequencing/ext_sequencing.py

```diff
@@ -184,7 +183,7 @@ def run(self):
         }
 
         for case in self._test_cases:
-            test_case_start = datetime.now()
+            test_case_start = random.choice(list(ssh_clients.values())).run_command("date '+%Y-%m-%d %T'").rstrip()
```

Comment:
Issues with time on orchestrator VM being different than the time on test ubuntu vms, as seen in other scenarios.

Using one of the VMSS instances to get the time on the test VMs